src/main/java/org/mule/extensions/jms/internal/config/JmsConfig.java

```diff
@@ -117,21 +117,29 @@ public Scheduler getResourceReleaserScheduler() {
 
   @Override
   public void dispose() {
-    if (scheduler != null) {
-      scheduler.stop();
+    synchronized (JmsConfig.class) {
```

Comment:
Please create a object look for this

Ej: private static final Object lock = new Object();
synchronized(lock)

Reply:
okey